src/crimson/osd/osd_operation.h

```diff
@@ -362,8 +360,6 @@ class OperationThrottler : public BlockerT<OperationThrottler>,
   uint64_t max_in_progress = 0;
   uint64_t in_progress = 0;
 
-  uint64_t pending = 0;
```

Comment:
Removing this is reasonable -- I think I meant this for debugging purposes but we can always readd it later.  Same comment as above though, it's unrelated to the other two changes and should be in its own commit (fine to include it as part of a larger PR though).

Reply:
I would handle it via separate PR.